deliveries/php-scale-lib_milestone_2.md

```diff
@@ -0,0 +1,17 @@
+# Milestone Delivery :mailbox:
+
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
+
+* **PR Link:** [php-scale-lib](https://github.com/w3f/Open-Grants-Program/pull/235)
```

Comment:
This is a bit more useful to us and conforms to the current [template](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md).

```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/php-scale-lib.md
```